roles/etcd/files/etcd.yaml

```diff
@@ -0,0 +1,40 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: master-etcd
+  namespace: kube-system
```

Comment:
I don't think we should re-use kube-system.  I'd like to separate based on compoenent.  We need to separate based on failure domains and startup ordering.  etcd has to come up before the kube-apiserver, so this needs its own namespace.

Reply:
Can you create static pods in another namespace?

Reply:
> Can you create static pods in another namespace?

Last I left it, yes.